go/vt/sqlparser/sql.y

```diff
@@ -245,6 +262,7 @@ func bindVariable(yylex yyLexer, bvar string) {
 %left <str> OR '|'
 %left <str> XOR
 %left <str> AND
+%left <str> WINDOW_EXPR
```

Comment:
Did you look at the precedence for Window_expr anywhere? Why did you add this in between the operators? The order in which symbols are specified has to do with the precedence of these operators.

Reply:
No, I didn't look for the precedence anywhere.
So, where should I reorder this value?

Reply:
Put it lower in the list where other tokens are defined